modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1430,7 +1430,7 @@ private static String getPrimitiveType(Schema schema) {
         } else if (ModelUtils.isDateTimeSchema(schema)) {
             return "DateTime";
         } else if (ModelUtils.isNumberSchema(schema)) {
-            if (schema.getFormat() == null) { // no format defined
+            if (schema.getFormat() == null || schema.getFormat().equals("decimal")) { // no format defined
```

Comment:
I suggest not to do this in this PR as this changing the defaul odegen will impact all other generators.

My recommendation is to discuss this further on how we support more formats.

Reply:
yes, actually this slipped in by mistake